### PR TITLE
MW-1168: additional changes in UI and mechanism in Adjustment

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openlmis.atlassian.net/browse/MW-1168

Changes: 
- improvements in UI for example bigger buttons (in NavBar), full width inputs/fields in forms, add indications that particular field is obligatory in form. 
- editable lot code (EditProductPage)\
- Toast for 'offline' mode and mechanism to show it while running offline mode. 
- Menu is not displayed on 'AddProductPage/EditProductPage'
- number input text aligned to right
- DateInput component

### CODE REVIEW

File: src/stock-adjustment-mobile/add-products-page/add-product-page.jsx [hunk 1]
```diff
@@ -163,7 +170,7 @@ const AddProductsPage = ({}) => {
         return (
             <SelectField
                 name={`${fieldName}.lot`}
-                label="Lot Code"
+                label="Lot Code*"
```
Comment (NikolaLaskowska) [inline src/stock-adjustment-mobile/add-products-page/add-product-page.jsx:173]: Why '*' is added? I cannot see the 'required' field. Can we make these required stars red as in mockups?

File: src/stock-adjustment-mobile/add-products-page/add-product-page.jsx [hunk 2]
```diff
@@ -204,7 +211,7 @@ const AddProductsPage = ({}) => {
                                                 <SelectField
                                                     required
                                                     name={`${name}.product`}
-                                                    label="Product"
+                                                    label="Product*"
```
Comment (NikolaLaskowska) [inline src/stock-adjustment-mobile/add-products-page/add-product-page.jsx:214]: Can we make these required stars red as in mockups?

Comment (sniedzielski) [reply src/stock-adjustment-mobile/add-products-page/add-product-page.jsx:214]: Yes. I can change this.

File: src/stock-adjustment-mobile/add-products-page/add-product-page.jsx (hunk 1 above)
Comment (sniedzielski) [reply src/stock-adjustment-mobile/add-products-page/add-product-page.jsx:173]: I will check this 'Lot Code' if this is obligatory or not. I can add 'red stars' as in mockups.

File: src/stock-adjustment-mobile/add-products-page/add-product-page.jsx (hunk 2 above)
Comment (sniedzielski) [reply src/stock-adjustment-mobile/add-products-page/add-product-page.jsx]: I added this change with 'red asterisk'

File: src/stock-adjustment-mobile/add-products-page/add-product-page.jsx (hunk 1 above)
Comment (sniedzielski) [reply src/stock-adjustment-mobile/add-products-page/add-product-page.jsx]: I added this change with 'red asterisk'

**Review by NikolaLaskowska — APPROVED** [review]

**Pull request merged**